WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingBottomSheetFragment.kt

```diff
@@ -97,6 +98,21 @@ class PrepublishingBottomSheetFragment : WPBottomSheetDialogFragment(),
                 behavior.peekHeight = metrics.heightPixels / 2
             }
         }
+        setupMinimumHeightForFragmentContainer()
+    }
+
+    private fun setupMinimumHeightForFragmentContainer() {
+        val isPage = checkNotNull(arguments?.getBoolean(IS_PAGE)) {
+            "arguments can't be null."
+        }
+
+        if (isPage) {
```

Comment:
Let me know your thoughts on this approach. I had to do this because the default minimum height that was set so the bottom sheet container remains at a good height no matter the fragment, was causing the page editor's sheet to have too much padding; the minimum height was too high so I lessened it for when it's a page vs a page. I realize this is some form of logic. I didn't run it through the `ViewModel` because it seemed a bit trivial. Nonetheless, happy to hear thoughts on this 🙏

Reply:
>I didn't run it through the ViewModel because it seemed a bit trivial. Nonetheless, happy to hear thoughts on this 🙏

Knowing that you considered running it through the VM and decided against it is enough ;)!

>was causing the page editor's sheet to have too much padding; the minimum height was too high so I lessened it for when it's a page vs a page. 

I'll test this on low end devices when I'm done reviewing the code ;).